package.json

```diff
@@ -9,7 +9,7 @@
     "test": "test"
   },
   "scripts": {
-    "deploy-prod": "hardhat run scripts/deploy.ts --network",
+    "deploy-prod": "hardhat run scripts/deployV1FactoryContractsts --network",
```

Comment:
I think you're missing a `.`?

Reply:
good catch, fixing it here https://github.com/BitGo/eth-multisig-v4/pull/175